src/loader/main.py

```diff
@@ -56,13 +62,17 @@ def main(endpoint):
 
     logger.info("STARTING: {}", endpoint["python_file"])
 
-    runner = importlib.import_module("endpoints.{}".format(endpoint["python_file"]))
+    try:
+        runner = importlib.import_module("endpoints.{}".format(endpoint["python_file"]))
+
+        data = runner.now(get_config(), force=True)
 
-    data = runner.now(get_config(), force=True)
+        if _test_data(data, runner.TESTS, endpoint):
 
-    if _test_data(data, runner.TESTS, endpoint):
+            _write_data(data, endpoint)
 
-        _write_data(data, endpoint)
+    except Exception as e:
+        logger.error("ERROR: {}", e)
```

Comment:
You actually should use `logger.exception(diagnose=True)`

see: https://loguru.readthedocs.io/en/stable/resources/recipes.html#customizing-the-formatting-of-exceptions

And, no need to add any message to it. The logger already know that it is an error and will print the full traceback.